server/config/config.go

```diff
@@ -69,6 +70,9 @@ const (
 	// DefaultEVMMinTip is the default minimum priority fee for the mempool
 	DefaultEVMMinTip = 0
 
+	// DefaultGethMetricsAddr is the default port for the geth metrics server.
+	DefaultGethMetricsAddr = `"127.0.0.1:8080"`
```

Comment:
weirdly this had to be wrapped in tick marks because otherwise it would write it without quotes and then the toml unmarshal would error.